src/main/java/com/github/wdstar/springboot/example/domain/RestClient.java

```diff
@@ -26,13 +26,11 @@ public RestClient(CircuitBreakerFactory circuitBreakerFactory, RestTemplate rest
 		this.rest = rest;
 	}
 
+	// default: maxAttempts = 3
+	@Retryable(value = { Throwable.class }, maxAttempts = 5)
 	public Map targetMethodWithCircuitBreaker() {
-		return circuitBreakerFactory.create("targetMethod").run(this.targetMethodSuppplier(), t -> {
-			logger.warn("restClient::targetMethod() call failed", t);
-			final Map<String, String> fallback = new HashMap<>();
-			fallback.put("hello", "circuit breaker fallback");
-			return fallback;
-		});
+		// without a fallback function.
+		return circuitBreakerFactory.create("targetMethod").run(this.targetMethodSuppplier());
```

Comment:
If this circuit breaker fails, an exception will be thrown, and then Spring Retry catches it and re-executes the method.